apps/bfd-server/bfd-server-war/src/main/java/gov/cms/bfd/server/war/stu3/providers/PatientResourceProvider.java

```diff
@@ -380,34 +386,41 @@ private String partDFieldFor(CcwCodebookVariable month) {
    * @return the criteria
    */
   private TypedQuery<Beneficiary> queryBeneficiariesBy(
-      String field, String value, PatientLinkBuilder paging, List<String> identifiers) {
-    String joinsClause = "";
+      String contractMonthField,
+      String contractYearField,
+      String partDContractId,
+      PatientLinkBuilder paging,
+      List<String> identifiers) {
+    String yearMonth = getYearMonth(contractYearField, contractMonthField);
+    String joinsClause = "inner join fetch b.enrollments ";
```

Comment:
You're not assigning this joined table an alias here, but down below, you're _assuming_ that you aliased it to "`e`". Not sure what the right syntax is for it, but you'll need to fix that.